common/components/configs/SectionLinkConfigs.js

```diff
@@ -3,17 +3,17 @@ import Section from '../enums/Section.js'
 const SectionLinkConfigs = [
   {
     section: Section.FindProjects,
-    title: 'FIND PROJECTS',
+    title: 'Find Projects',
```

Comment:
Thank you for doing this bit of cleanup. Good to do, if we want all caps in the DOM we can use text-transform CSS rules instead. More readable code = better. :)